paws/templates/mediawiki-hacks.yaml

```diff
@@ -1,43 +1,53 @@
 {{ if .Values.mediawiki.enabled }}
 # We assume mediawiki is only enabled in a local dev environment, so...
-apiVersion: batch/v1
-kind: Job
+---
+apiVersion: v1
+kind: PersistentVolume
```

Comment:
This is a quirk fo the Minikube volume provisioner. A 'persistent volume' is actually a host path. Only the one under /data is actually persistent, but the other paths just need to exist to keep some pods happy. Only the userhomes volume matters. I'm hoping the docker driver won't block the mount in some way that requires a VM.